script/core.hbs

```diff
@@ -73,14 +73,32 @@
          * Determine whether a URL is absolute or not.
          * Common examples: "mailto:slapshot@gmail.com", "//yext.com", "https://yext.com"
          */
-        ANSWERS.registerHelper('isNonRelativeUrl', function(str) {
+        function isNonRelativeUrl(str) {
```

Comment:
It's kind of crummy that we have to basically duplicate `isNonRelativeUrl` and `relativePathUrlHandler`. It'd be great if the JS in `hbshelpers` could be inlined as a partial here. Would save us having to update the same code-block twice. Is there any non-janky way to do that?

Reply:
You know, at first I thought this was going to be hard because it's hard to get the path from entry.js to the hbshelpers folder, since than depends on the jambo config, and we don't want to use dynamic imports here (with like a require.resolve or something) BUT I'm remembering now that webpack lets you add resolve aliases. Will add in a separate PR.

Reply:
I lied. Webpack doesn't like it when you mix ES6 and commonjs, and unfortunately jambo only understands hbs helpers that are commonjs. So putting it into the bundle is probably out of the window for now.
I tried thinking about how to do the inlining method as well with just HBS, but obviously module.exports is not something hbs understands. We could do jank where we create an empty object called "module", then loop over the desired helpers, continually reassigning the value of module.exports and calling ANSWERS.registerHelper on it. jank !== not janky though

Reply:
**sad developer noises** this was the pr https://github.com/yext/answers-hitchhiker-theme/pull/871